zeppelin-interpreter/src/test/java/org/apache/zeppelin/user/AuthenticationInfoTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.zeppelin.user;
+
+import static org.junit.Assert.assertEquals;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Test;
+
+public class AuthenticationInfoTest {
+
+  @Test
+  public void testRoles() {
+    final String roles = "[role1, role2, role with space]";
+
+    final AuthenticationInfo authenticationInfo = new AuthenticationInfo("foo",
+        roles, "bar");
+
+    assertEquals(authenticationInfo.getRoles(),
+        new ArrayList<>(Arrays.asList("role1", "role2", "role with space")));
```

Comment:
Minor. Switch left and right. The left should be expected, and right is actual.

Reply:
Good catch. Fixed :+1: